package-res/resources/web/pentaho/data/Filter.js

```diff
@@ -44,7 +110,7 @@ define([
 
     /**
      * Create a filter from a spec
-     * @param {Object} spec - Specification of a Filter
+     * @param {pentaho.data.filter.Or} spec - Specification of a Filter
      * @returns {*}
      */
```

Comment:
Wrong type. Something like:
@param {Object} spec - Specification of a Filter
@return {pentaho.data.filter.Or} The filter